test/TestFramework.ForTestingMSTest/AdapterToTestPlatform.cs

```diff
@@ -128,6 +128,19 @@ private static void LogMessage(IMessageLogger? logger, TestMessageLevel level, s
         }
     }
 
+    // Looking up base types.
+    private static bool IsAnyBaseClassInheritingTestContainer(Type typeInfo)
```

Comment:
I would recommend renaming this into either `IsInheritingTestContainer` or `IsTestContainer`.